README.md

```diff
@@ -11,17 +11,23 @@ Any contribution is welcome!
 
 * [The Gimp](https://www.gimp.org) - Image editor 
 * [Krita](https://krita.org) - Image editor 
-* [Opentoonz](https://opentoonz.github.io/) - Animation production
+* [Opentoonz](https://opentoonz.github.io/) - Animation
+* [Synfig](https://www.synfig.org/) - Animation 
+* [Pencyl2D](https://www.pencil2d.org/) - Hand-drawn animation
```

Comment:
Little typo there, should be "Pencil2D" (no `y`)

Reply:
Oups you're right!

Reply:
I fixed it after: https://github.com/cgwire/awesome-cg-vfx-pipeline/commit/3278f75c44d966c23b25b6be3a6fc7789c003206